pkg/lib/bundle/build.go

```diff
@@ -50,15 +50,14 @@ func ExecuteCommand(cmd *exec.Cmd) error {
 // @channels: The list of channels that bundle image belongs to
 // @channelDefault: The default channel for the bundle image
 // @overwrite: Boolean flag to enable overwriting annotations.yaml locally if existed
-func BuildFunc(directory, outputDir, imageTag, imageBuilder, packageName, channels, channelDefault string,
-	overwrite bool) error {
+func BuildFunc(directory, outputDir, imageTag, imageBuilder, packageName, channels, channelDefault string, validateCSV, overwrite bool) error {
```

Comment:
Note for future refactoring: you can tell from the ballooning inputs here that there will be lots of branching in this function. 

CSV validation might make more sense as something to run over the output of this function rather than a flag that is passed in. 

Overwrite might make more sense as a Writer Interfaxe that gets set to a Noop if the overwrite flag is set to false, etc.